src/Avalonia.DesignerSupport/Remote/FileWatcherTransport.cs

```diff
@@ -70,16 +69,24 @@ async void UpdaterThread()
         {
             while (!_disposed)
             {
-                var data = File.ReadAllText(_path);
-                if (data != _lastContents)
+                try
```

Comment:
What warning is this fixing? I wasn't aware of any warnings related to missing `try` blocks?

Reply:
Remote\FileWatcherTransport.cs(62,76): warning CS0067: The event 'FileWatcherTransport.OnException' is never used

Reply:
This change changes the control flow, not approving as a part of "fix warnings" PR

Reply:
I reverted it